pkg/apis/options/legacy_options.go

```diff
@@ -488,6 +488,8 @@ type LegacyProvider struct {
 	KeycloakGroups                         []string `flag:"keycloak-group" cfg:"keycloak_groups"`
 	AzureTenant                            string   `flag:"azure-tenant" cfg:"azure_tenant"`
 	AzureGraphGroupField                   string   `flag:"azure-graph-group-field" cfg:"azure_graph_group_field"`
+	MSEntraIDSkipGroupsFromGraph           bool     `flag:"ms-entra-id-skip-groups-from-graph" cfg:"ms_entra_id_skip_groups_from_graph"`
+	MSEntraIDMultiTenantAllowedTenants     []string `flag:"ms-entra-id-multi-tenant-allowed-tenant" cfg:"ms_entra_id_multi_tenant_allowed_tenants"`
```

Comment:
Instead of adding the booling at the bottom cannot we just check if the list is empty or not?

or is the list optional?

```suggestion
	EntraIDAllowedTenants     []string `flag:"entra-id-allowed-tenant" cfg:"ms_entra_id_multi_tenant_allowed_tenants"`
```

Reply:
List is optional, by default accept all tenants when using multi-tenant apps